apps/mark-scan/backend/src/app.test.ts

```diff
@@ -546,96 +548,116 @@ function expectVotesEqual(expected: VotesDict, actual: VotesDict) {
   expect(sortVotes(actual)).toEqual(sortVotes(expected));
 }
 
-test('printing ballots', async () => {
-  const printBallotSpy = vi.spyOn(stateMachine, 'printBallot');
-
-  const deferredEjection = deferred<boolean>();
-  const mockEject = vi.spyOn(driver, 'ejectBallotToRear');
-  mockEject.mockReturnValue(deferredEjection.promise);
-
-  const electionDefinition = getMockMultiLanguageElectionDefinition(
-    readElectionGeneralDefinition(),
-    ['en', 'zh-Hans']
-  );
-  await configureForTestElection(
-    electionDefinition,
-    electionGeneralFixtures.uiStrings
-  );
-
-  await expectElectionState({ ballotsPrintedCount: 0 });
-
-  // vote a ballot in English
-  await mockLoadFlow(apiClient, driver);
-  const mockVotes = generateMockVotes(electionDefinition.election);
-
-  // Add extreme-case write-in to track handling of long names with no breaks.
-  const maxWriteInChars = 40 - 'WRITEIN'.length;
-  const fillCharCount = maxWriteInChars - 'WRITEIN'.length;
-  const writeInName = `WRITEIN${'N'.repeat(fillCharCount)}`;
-
-  const writeInContest = find(
-    electionDefinition.election.contests,
-    (c) => c.type === 'candidate' && c.allowWriteIns && c.seats === 1
-  );
-  mockVotes[writeInContest.id] = [
-    {
-      id: `write-in-${writeInName}`,
-      isWriteIn: true,
-      name: writeInName,
-    },
-  ];
-
-  await apiClient.printBallot({
-    precinctId: '21',
-    ballotStyleId: electionDefinition.election.ballotStyles.find(
-      (bs) => bs.languages?.includes('en')
-    )!.id,
-    votes: mockVotes,
-    languageCode: 'en',
-  });
-
-  await expectElectionState({ ballotsPrintedCount: 1 });
-  const pdfData = printBallotSpy.mock.calls[0][0];
-  await expect(pdfData).toMatchPdfSnapshot({ failureThreshold: 0.027 });
-
-  await waitForStatus('presenting_ballot');
-  const interpretation = await apiClient.getInterpretation();
-  assert(interpretation);
-  assert(interpretation.type === 'InterpretedBmdPage');
-  expectVotesEqual(interpretation.votes, mockVotes);
+test.each([
```

Comment:
Review this diff using https://github.com/votingworks/vxsuite/pull/6514/commits/0d69f55afed87f03cc3bbe95ccf931c1f0627973?w=1